avocado/core/test.py

```diff
@@ -961,15 +957,14 @@ def test(self):
         raise exceptions.NotATestError(e_msg)
 
 
-class SkipTest(Test):
+class FakeTest(Test):
```

Comment:
I thought we wanted to completely avoid using `Test` class as it's inappropriate. How about creating `BaseTest` class with only the essential stuff (without `data_dir` and such) and use it as a base for `Test` and other dummy classes?